src/libpgagroal/pipeline_transaction.c

```diff
@@ -544,8 +542,8 @@ transaction_server(struct ev_loop* loop, struct ev_io* watcher, int revents)
    errno = 0;
 
    exit_code = WORKER_CLIENT_FAILURE;
-   running = 0;
-   ev_break(loop, EVBREAK_ALL);
+
```

Comment:
empty line, this patter repeats, either keep all the empty lines or remove (as was before)